doc/user/content/manage/reverse-etl/census.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Census"
+description: "How to use Census to export data out of Materialize."
+menu:
+  main:
+    parent: "reverse-etl"
+---
+
+This guide walks you through the steps required to create a [Census](https://www.getcensus.com/) sync using Materialize.
+
+## Before you begin
+
+In order to build a sync with Census you will need:
+
+* A table, view, materialized view or source within your Materialize account that you would like to export.
+* A [Braze](https://www.braze.com/) account. Census supports a number of possible [destinations](https://www.getcensus.com/integrations), we will use Braze as an example.
+
+## Step 1. Set up a Materialize Source
+
+To begin you will need to add your Materialize database as a source in Census.
+
+1. In Census, navigate to **Sources** and then click **New Source**.
+
+1. From the list of connection types, choose **Materialize**.
+
+1. Set the connection parameters using the credentials provided in the [Materialize console](https://console.materialize.com/).
```

Comment:
I wouldn't say so (I like other approaches in some contexts like the one in Deepnote) but I favor for using the same structure rather than having different structures around.